src/builder.rs

```diff
@@ -1335,15 +1476,16 @@ impl<'ctx> Builder<'ctx> {
     ///
     /// assert!(module.verify().is_ok());
     /// ```
-    pub fn build_bitcast<T, V>(&self, val: V, ty: T, name: &str) -> BasicValueEnum<'ctx>
+    pub fn build_bitcast<T, V>(&self, val: V, ty: T, name: &str) -> AnyValueEnum<'ctx>
```

Comment:
Why was this changed to any?

Reply:
I changed it to be able to use it on function pointers, e.g. cast them to some primitive type like i256.

Reply:
FunctionValues can be converted to PointerValues, so I think we should leave it as is

Reply:
Found it. Gonna change back.